extensions/Worker.Extensions.Abstractions/src/MetadataBindingPropertyName.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.Abstractions
+{
+    /// <summary>
+    /// Specifies the binding property name that is used when generating function metadata.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Property, AllowMultiple = false)]
+    public sealed class MetadataBindingPropertyName : Attribute
```

Comment:
nit: [naming] This is missing the `Attribute` suffix. With that (or without) the name feels a bit long. Given how this is used, I think shortening to `BindingPropertyName` would be appropriate, so effectively `BindingPropertyNameAttribute`

Reply:
Fixed the attribute name to "BindingPropertyNameAttribute" as suggested. Thank you!